fix(clustered): remove link to inaccessible google doc (#5197)
influxdata/docs-v2: content/influxdb/clustered/admin/users/_index.md

draft: true
---

To add or remove users, update the users list in the `myinfluxdb.yml` file.
The users list is found at spec.package.spec.admin.users.
After updating the list, re-apply `myinfluxdb.yml`.

> To learn how to apply `myinfluxdb.yml`, see [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
> After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
> When the updates are finished, new users will have been added, and removed users will have been deleted.```

Reviewer: ```suggestion
To learn how to apply `myinfluxdb.yml`, see [Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
When the updates are finished, new users will have been added, and removed users will have been deleted.
```

> To add or remove users, update the users list in `myinfluxdb.yml` file.

Reviewer 2: ```suggestion
To add or remove users, update the users list in the `myinfluxdb.yml` file.
```

The users list is found at spec.package.spec.admin.users.
After updating the list, re-apply `myinfluxdb.yml`.

> See [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy) for details on how to apply `myinfluxdb.yml`.

Reviewer 2: ```suggestion
To learn how to apply `myinfluxdb.yml`, see [Step 6: Deploy an InfluxDB cluster](/influxdb/clustered/install/deploy).
```

> Once the `myinfluxdb.yml` has been applied, it will take a couple minutes for the updates to roll out.
> When complete, any new users will have been added and any removed users will have been deleted.

Reviewer 2: ```suggestion
After `myinfluxdb.yml` has been applied, updates take a couple of minutes to complete.
When the updates are finished, new users will have been added, and removed users will have been deleted.```
